src/Datadog.Trace/Conventions/IOutboundHttpConvention.cs

```diff
@@ -8,5 +8,7 @@ namespace Datadog.Trace.Conventions
     internal interface IOutboundHttpConvention
     {
         Scope CreateScope(OutboundHttpArgs args, out HttpTags tags);
+
+        TraceId GenerateNewTraceId();
```

Comment:
This should be removed per https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/72/files#r596037493

Reply:
Done, moved to separate interface for trace id conventions.